src/partials/layout.handlebars

```diff
@@ -66,16 +66,22 @@
                         <li><a href="/events-reference/events-reference-serenity/products.html">Events API Reference</a></li>
                     </ul>
                 </li>
+                <li class="{{#if active_guided_tutorials}}active{{/if}}">
```

Comment:
I think you can remove the active_guided_tutorials display condition: it always be displayed.